test/metabase/db/custom_migrations_test.clj

```diff
@@ -240,10 +240,10 @@
 
 (deftest downgrade-dashboard-tabs-test
   (testing "Migrations v47.00-029: downgrade dashboard tab test"
-    (impl/test-migrations ["v47.00-029"] [_migrate!]
+    (impl/test-migrations ["v47.00-029", "v47.00-030"] [migrate!]
```

Comment:
if we don't have a stopping point for this test, the cards in this test will be migrated to 24 as well, and the rollback result will change. and since the migration from grid 18 to 24 is not idempotence, it makes this test less isolated.

so it's easier to just isolate it.

Reply:
bump

Reply:
actually if we want to only run a specific version we should pass a string only, not a seq.
like this `(impl/test-migrations "v47.00-029" ...)`, doing `(impl/test-migrations ["v47.00-029"] ...)` will run from from `v47.00-029` through end. see https://github.com/metabase/metabase/blob/0b27bb92ba856adc0e286894d45d941eec0114e4/test/metabase/db/schema_migrations_test/impl.clj#L212

Reply:
I used the single string here 4f169ca09c